_faq/cep-and-its-importance.md

```diff
@@ -0,0 +1,25 @@
+---
+question: "What is a cEP? Why is it so important to do a pre-GSoC analysis
+and design a proposal as early as possible?"
+---
+A **cEP (coala Enhancement Proposal)** is proposed to the **coala** community
+to discuss the new *major features* that the contributor intends to implement
+during the GSoC duration. The proposal is designed before the implementation of
+*major features* because the implementation usually happens from a technical
+perspective which may lead to a poorly designed content, both developer-wise
+and user-wise. So, to avoid this, a cEP should be designed at an earlier stage
+and this essentially saves a lot of coding work and makes sure that major
+features are properly documented.
+
+Experience tells us that the best GSoC students are the ones who heavily
+contribute to the project ideas' website, create projects from the issues, and
+expand on the existing ***.md*** files.
+
+So, if you are planning to participate in GSoC with coala, you should start
+writing a detailed project idea in our [projects](https://github.com/coala/projects)
+repository. Also, you must do enough research about the project in which you
+are interested, so that you're able to express your ideas, timeline, etc
```

Comment:
There should be some punctuation after `etc`, no?

Reply:
I was thinking the same thing but apparently, both can be used as per the Wikipedia :sweat_smile: 
Could still be changed though.